src/e2e-test/features/bigquery/sink/GCSToBigQuery.feature

```diff
@@ -0,0 +1,33 @@
+@BigQuery_Sink
+Feature: BigQuery sink - Verification of GCS to BigQuery successful data transfer
+
+  @CMEK @GCS_CSV_TEST @BQ_TARGET_TEST
+  Scenario:Validate successful records transfer from GCS to BigQuery
+    Given Open Datafusion Project to configure pipeline
+    When Source is GCS
+    When Sink is BigQuery
+    Then Open GCS source properties
+    Then Enter the GCS source mandatory properties
+    Then Validate "GCS" plugin properties
+    Then Close the GCS properties
+    Then Open BigQuery sink properties
+    Then Enter BigQuery property reference name
+    Then Enter BigQuery properties projectId "projectId" and datasetProjectId "projectId"
+    Then Enter BigQuery sink property dataset "dataset" and table name
+    Then Enter BiqQuery property encryption key name if cmek is enabled
```

Comment:
Similar comment pass encryption key name as argument to the step definition.

Reply:
Done